CHANGELOG.md

```diff
@@ -285,6 +285,7 @@ queries issued before 5.0 will no longer cascade query terminiation.
 * abs, ceil and floor will now return types aligned with
 other databases systems (i.e. the same type as the input). Previously
 these udfs would always return Double.
+* Statements in the command topic will be retried until they succeed. For example, if the source topic has been deleted for a create stream/table statement, the server may fail to start since it will be stuck processing the statement. This ensures that the same set of streams/tables are created when restarting the server.
```

Comment:
Maybe also add a note about how to tell if the command runner thread is stuck? JMX metrics, logs, etc.

Reply:
added